etl/steps/data/garden/itopf/2023-05-18/oil_spills.py

```diff
@@ -0,0 +1,68 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import numpy as np
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("oil_spills.start")
+    print("hi")
```

Comment:
Oh, hi! 😃

Reply:
haha! forgot about this! I will do some final checks (esp metadata) and then squash and merge :) thanks so much again for all your help! I am sorry I've given you so much unexpected work! but I did learn a lot from this one haha :D